src/ApiCore/GapicClientTrait.php

```diff
@@ -197,12 +354,14 @@ private function startOperationsCall(
      *
      * @return PagedListResponse
      */
-    private function getPagedListResponse(Call $call, array $options, array $descriptor)
+    private function getPagedListResponse(Call $call, array $optionalArgs, array $descriptor)
```

Comment:
Should this be `startPagedUnaryCall`? But it isn't starting a call, it is finishing it... maybe `makePagedUnaryCall`? I'm not sure